doc/authors.rst

```diff
@@ -51,6 +51,8 @@ contribution, and may not be the current affiliation of a contributor.
 * Florian Porrmann [13]
 * Sarah Pilz [13]
 * Oliver Kloß [1]
+* Jonas Oberste-Frielinghaus [1]
+* Sven Krausse [1]
```

Comment:
Are Jonas and Sven aware and in agreement of the new approach to change `t_stop`?
Sorry, I might have missed or forgotten this part.
I do renember they have been involved with the first version of the suggested fix.

The affilaition of Jonas and you chnaged to IAS-6, refer to `authors.rst` in v1.1.0, reference [14].

https://github.com/NeuralEnsemble/elephant/blob/15e8ef8e2a3b9853f9d7ca6bffd158c56cb545ba/doc/authors.rst?plain=1#L69